fares-data-build-tool/fdbt-site
# feat: school service fare type selection

# Description

-   This PR adds a new page to the school tickets user journey, allowing a user to select a secondary fare type.

# Testing instructions

-   Pull down this branch and run the site locally. Validate that all existing journeys on the site work as they should and that the new page works as expected.

# Type of change


# Checklist:
